### DISCUSSION TITLE
Registre des consentements

### DISCUSSION THREAD

**Discussion by myalban** [AmauriC/tarteaucitron.js#1188] [General]
Salut, est ce que le registre des consentements est obligatoire ?
je ne vois pas d'option dans TAC pour l'enregistrement de ces informations vers une base de données

Merci
Alban

**Comment by AmauriC** [comment]
Salut,
Non, ce n'est pas obligatoire, il y a d'autres méthodes qui n'imposent pas de faire un traitement de donnée supplémentaire.
Voir la fin de ce guide: https://www.cnil.fr/fr/cookies-et-autres-traceurs/regles/cookies/que-dit-la-loi

**Reply by myalban** [reply]
Merci pour cette explication.
